torchvision/prototype/transforms/functional/_augment.py

```diff
@@ -1,52 +1,57 @@
-from typing import Tuple
+from typing import TypeVar, Any
 
 import torch
+from torchvision.prototype import features
+from torchvision.prototype.transforms import kernels as K
 from torchvision.transforms import functional as _F
 
+from ._utils import dispatch
 
-erase_image = _F.erase
+T = TypeVar("T", bound=features.Feature)
 
 
-def _mixup(input: torch.Tensor, batch_dim: int, lam: float, inplace: bool) -> torch.Tensor:
-    if not inplace:
-        input = input.clone()
+@dispatch(
+    {
+        torch.Tensor: _F.erase,
+        features.Image: K.erase_image,
+    }
+)
+def erase(input: T, *args: Any, **kwargs: Any) -> T:
+    """ADDME"""
+    ...
 
-    input_rolled = input.roll(1, batch_dim)
-    return input.mul_(lam).add_(input_rolled.mul_(1 - lam))
 
+@dispatch(
+    {
+        features.Image: K.mixup_image,
```

Comment:
I believe we are missing the dispatch for `torch.Tensor`. Worth checking where else this is the case and add it.